tests/test_sageintacct/test_tasks.py

```
@@ -2108,3 +2108,143 @@
+    )
+
+    corporate_card = ExpenseAttribute.objects.create(
+        attribute_type='CORPORATE_CARD',
+        value='American Express - 61662',
+        display_name='Corporate Card',
+        source_id='baccjpfvrtsPg9',
+        workspace_id=1,
+        active=True
+    )
+
+    mapping = Mapping.objects.create(
```

Ruff (0.8.2)

Local variable `mapping` is assigned to but never used

Remove assignment to unused variable `mapping`

(F841)